src/Azure.Functions.Cli/ExtensionBundle/ExtensionBundleHelper.cs

```diff
@@ -6,9 +6,8 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging.Abstractions;
 using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Text;
+using static System.Environment;
```

Comment:
This is used for SpecialFolder? If yes, please use Environment.SpecialFolder to be more explicit